articles/openshift/howto-using-azure-redhat-openshift.md

```diff
@@ -132,9 +132,14 @@ Follow these steps to create a virtual network containing two empty subnets.
 
 ## Create a cluster
 
-Run the following command to create a cluster.
+Run the following command to create a cluster. 
+
+* The '-g $RESOURCEGROUP' parameter refers to the OpenShift resource and can be used by multiple clusters.
+* The '--cluster-resource-group "aro-$CLUSTER' parameter referes to a new resource group that will be created for this specific cluster.
```

Comment:
I would add a note, that it is the group where VMs and all the resources residde.